.github/workflows/ci_cd.yml

```diff
@@ -169,13 +169,13 @@ jobs:
       contents: write
     steps:
       - name: Release to the public PyPI repository
-        uses: ansys/actions/release-pypi-public@v8
+        uses: ansys/actions/release-pypi-public@v9
         with:
          use-trusted-publisher: true
          library-name: ${{ env.PACKAGE_NAME }}
 
```

Comment:
@AlejandroFernandezLuces -- please adapt to use the new action. Have a look at the actions migration guide

* https://actions.docs.ansys.com/version/stable/migrations/index.html#version-v9-0
* https://actions.docs.ansys.com/version/stable/migrations/release-pypi-trusted-publisher.html#release-pypi-trusted-publisher